internal/processing/processors/virtual_service_processor_test.go

```diff
@@ -78,6 +84,65 @@ var _ = Describe("Virtual Service Processor", func() {
 	})
 })
 
+var _ = Describe("GetVirtualServiceHttpTimeout", func() {
+	It("should return default when no timeout is set", func() {
+		// given
+		apiRuleSpec := gatewayv1beta1.APIRuleSpec{}
+		rule := gatewayv1beta1.Rule{}
+
+		// when
+		timeout := processors.GetVirtualServiceHttpTimeout(apiRuleSpec, rule)
+
+		// then
+		Expect(timeout).To(Equal(time.Second * 180))
```

Comment:
The 180 default is not really known from the test's perspective.
Maybe adjust the description to `should return default 180s when no timeout is set`?

Or we should keep the timeout in helpers package and share it across all files that use it

Reply:
I'd not keep it in the helpers package, since it's only relevant for the virtual service. So the reason why I moved it closer to the virtual service is cohesion and slowly removing the helper package.

Reply:
I'm not a big fan of re-using constant from real code in tests or exporting them for test only, since changing this values often is a break of the functional contract of your code. If we change the value it should break some tests, to make it obvious that some external behaviour changed. If it's exported it can be easily overseen during refactoring.